packages/ui5-application-writer/test/defaults.test.ts

```diff
@@ -15,12 +15,13 @@ describe('Defaults', () => {
         ['1.80-snapshot', esmTypesVersionSince],
         ['metadata', typesVersionBest],
         [undefined, typesVersionBest],
-        ['1.109.1', typesVersionBest],
+        ['1.109.1', '~1.109.0'],
         [UI5_DEFAULT.TYPES_VERSION_BEST, typesVersionBest],
-        ['1.109-snapshot', typesVersionBest],
+        ['1.109-snapshot', '~1.109.0'],
         ['1.80-snapshot', esmTypesVersionSince],
-        ['1.102-snapshot', '~1.102.0'],
-        ['1.91.0', '~1.91.0']
+        ['1.102-snapshot', typesVersionBest],
+        ['1.91.0', '~1.91.0'],
+        [UI5_DEFAULT.UI5_VERSION_VARIABLE, 'latest']
```

Comment:
Currently 'latest' for [@sapui5/ts-types-esm](https://www.npmjs.com/package/@sapui5/ts-types-esm?activeTab=versions) is `1.108.4` and not the available `1.109.3`
<img width="828" alt="Screenshot 2022-12-20 at 11 58 52" src="https://user-images.githubusercontent.com/10102054/208661946-d6202c43-8b2b-4253-97aa-c9e22836b0a0.png">

@petermuessig - does `latest` point to latest LTS version only and not the highest available version normally?

Reply:
I think we can opt for `>=1.109.1` if the minUI5Version is `${sap.ui5.dist.version}`